PrototopeJSBridge/GeometryBridge.swift

```diff
@@ -173,6 +174,31 @@ import Prototope
 		self.rect = rect
 		super.init()
 	}
+    
+    public func inset(args: NSDictionary) -> RectJSExport {
+        let hasKey = { (key: String) -> Bool in
+            return (args.objectForKey(key) as? Double) != nil
+        }
+        let getValue = { (key: String) -> Double in
+            return (args.objectForKey(key) as! Double?) ?? 0
+        }
+        
+        let justValue = hasKey("value")
+        let simplified = hasKey("vertical") || hasKey("horizontal")
+        let core = hasKey("top") || hasKey("right") || hasKey("bottom") || hasKey("left")
+        
+        switch (justValue, simplified, core) {
```

Comment:
Ah, the point here was to make sure that users of this API don’t try to mix and match the possible sets arguments.